### PR TITLE
Add create detector endpoint

### LINKED ISSUE #14: Create a detector

### PR DESCRIPTION
**Pull request by ao-picterra**
Closes #14.

### CODE REVIEW

File: src/picterra/client.py [hunk 1]
```diff
@@ -175,6 +175,53 @@ def _is_ready():
                 return False
         _poll_at_interval(_is_ready, poll_interval)
 
+    def add_raster_to_detector(self, raster_id, detector_id):
+        """
+        Associate a raster to a detector
```
Comment (julienr) [inline src/picterra/client.py:180]: Can we just mention this is beta ?

File: src/picterra/client.py [hunk 2]
```diff
@@ -175,6 +175,53 @@ def _is_ready():
...
+            data={
+                'raster_id': raster_id
+            }
+        )
+        if not resp.status_code == 201:
+            raise APIError(resp.text)
+
+    def create_detector(self, name: str = '', type: str = 'count') -> dict:
+        """
+        Associate a raster to a detector
```
Comment (julienr) [inline src/picterra/client.py:200]: Can we just mention this is beta ?

**Review by julienr — COMMENTED** [review]
LGTM - I would just mention in the function docstrings that training-related things are beta

File: src/picterra/client.py (hunk 2 above)
Comment (ao-picterra) [reply src/picterra/client.py:202]: Checked

File: src/picterra/client.py (hunk 1 above)
Comment (ao-picterra) [reply src/picterra/client.py:180]: Checked

**Review by julienr — APPROVED** [review]

**Pull request merged**